src/desktop/users.desktop

```diff
@@ -9,7 +9,7 @@ X-SuSE-YaST-Call=users
 X-SuSE-YaST-Group=Security
 X-SuSE-YaST-Argument=
 X-SuSE-YaST-RootOnly=true
-X-SuSE-YaST-AutoInst=all
+X-SuSE-YaST-AutoInst=configure
```

Comment:
Ok. And as the user keys are removed from the profile before the second stage, the users_auto will do anything. I've tried manually and it works.